lib/charms/kratos_external_idp_integrator/v0/kratos_external_provider.py

```diff
@@ -295,7 +303,7 @@ def parse_config(cls, config):
         ]
 
     @classmethod
-    def _parse_provider_config(cls, config):
+    def _parse_provider_config(cls, config: Dict) -> Dict:
```

Comment:
I think at least for now it will return `None`
```suggestion
    def _parse_provider_config(cls, config: Dict) -> None:
```

Reply:
The actual implementations of this method return `Dict`. If I change this to `None`, `mypy` will complain.